.directory

```diff
@@ -0,0 +1,6 @@
+[Dolphin]
```

Comment:
What is .directory?

Reply:
.directory is file default in dophin on kde. =/

Reply:
remove this please.